pkg/cassandra/datacenter.go

```diff
@@ -410,6 +415,10 @@ func AddInitContainersToPodTemplateSpec(dcConfig *DatacenterConfig, initContaine
 		return errors.New("PodTemplateSpec was nil, cannot add init containers")
 	} else {
 		dcConfig.PodTemplateSpec.Spec.InitContainers = append(dcConfig.PodTemplateSpec.Spec.InitContainers, initContainers...)
+		position, found := FindInitContainer(dcConfig.PodTemplateSpec, "server-config-init")
```

Comment:
In the `reconciliation` package of cass-operator, there is an exported constant for this container's name: `reconciliation.ServerConfigContainerName`. It would be slightly better to use it instead of hard-coding the name.